WalletWasabi.Fluent/Styles/Styles.xaml

```diff
@@ -15,4 +15,7 @@
     <Setter Property="IsPaneOpen" Value="True" />
     <Setter Property="PaneBackground" Value="{x:Null}" />
   </Style>
+  <Style Selector="TextBlock">
+    <Setter Property="Foreground" Value="{DynamicResource TextForeground}"></Setter>
```

Comment:
nit: `<Setter />` not `<Setter></Setter>` 

if the content of a tag is empty.

Reply:
Done